### PR TITLE
fix: Remove location from client activation info (WEBAPP-5518)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/view_model/content/PreferencesDevicesViewModel.js [hunk 1]
```diff
@@ -90,11 +70,8 @@ z.viewModel.content.PreferencesDevicesViewModel = class PreferencesDevicesViewMo
 
   updateDeviceInfo() {
     if (this.currentClient() && !this.localFingerprint().length) {
-      const {location, time} = this.currentClient();
+      const {time} = this.currentClient();
```
Comment (atomrc) [inline app/script/view_model/content/PreferencesDevicesViewModel.js]: You probably don't need a tmp variable for that. The two lines should fit into one

File: app/script/view_model/content/PreferencesDeviceDetailsViewModel.js [hunk 2]
```diff
@@ -37,56 +37,32 @@ z.viewModel.content.PreferencesDeviceDetailsViewModel = class PreferencesDeviceD
     this.conversationRepository = repositories.conversation;
     this.cryptographyRepository = repositories.cryptography;
     this.logger = new z.util.Logger('z.viewModel.content.PreferencesDeviceDetailsViewModel', z.config.LOGGER.OPTIONS);
-    this.locationRepository = repositories.location;
```
Comment (atomrc) [inline app/script/view_model/content/PreferencesDeviceDetailsViewModel.js:40]: Is the locationRepo used somewhere else? Maybe we can totally remove it?

Comment (AndyLnd) [reply app/script/view_model/content/PreferencesDeviceDetailsViewModel.js:40]: It is still used.

Comment (gregor) [reply app/script/view_model/content/PreferencesDeviceDetailsViewModel.js:40]: It should not be used anywhere. We can remove geolocation and CSP headers related to it.

**Comment by gregor** [comment]
Do we display the location on the auth pages as well?

**Comment by AndyLnd** [comment]
We don't display it on the auth pages.

**Comment by gregor** [comment]
Why against the edge branch though? This is unrelated to the new feature and should show up in the commit history after a squash merge from the edge branch.

**Pull request merged**